{"name": "test_with_issue_link", "status": "passed", "start": 1587114095476, "stop": 1587114095476, "uuid": "613f1c42-fd81-43a6-84b7-de6f797333d2", "historyId": "e21b7c934dee72527417c5f147526411", "testCaseId": "e099b2fe0344ed9be1f2c29527e05911", "fullName": "test_link#test_with_issue_link", "labels": [{"name": "suite", "value": "test_link"}, {"name": "host", "value": "pinrongdeMacBook-Pro.local"}, {"name": "thread", "value": "5044-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_link"}], "links": [{"type": "issue", "url": "140", "name": "Pytest-flaky test retries shows like test steps"}]}